draft-bbs-signatures.md

```diff
@@ -738,6 +748,76 @@ Procedure:
 5. return (scalar_1, ..., scalar_n)
 ```
 
+### OctetsToSignature
+
+This operation describes how to decode an octet string, validate it and return the underlying components that make up the signature.
+
+```
+(A, e, s) = octets_to_signature(signature_octets)
+
+Inputs:
+
+- signature_octets, octet string.
+
+Parameters:
+
+- octet_point_length, the length in bytes of the octet string output by the point_to_octets function
+
+Outputs:
+
+- A map comprised of the underlying signature components `A` `e` and `s`, OR the result INVALID
+
+Procedure:
+
+1. if len(signature_octets) != (octet_point_length + 2 * xof_no_of_bytes), return INVALID
```

Comment:
This is incorrect, the compressed size of the scalar needs to be captured and denoted correctly.